terraform-use-modules/step2.md

```diff
@@ -0,0 +1,139 @@
+In this step, you will set and define the module input variables.
+
+## Review values for module input variables
```

Comment:
```suggestion
## Review Values for Module Input Variables
```

Other scenarios use this style for headings - but the learn style guide says sentence style. Not sure which one is best for katacoda, but we should stick with one.

Reply:
I think most of them have been sentence case. Ill update them all to sentence case :)